appserver/extras/embedded/web/pom.xml

```diff
@@ -289,6 +290,14 @@
            <type>zip</type>
            <optional>true</optional>
        </dependency>
+
+        <dependency>
+          <groupId>fish.payara.packager</groupId>
+          <artifactId>payara-api-package</artifactId>
+          <version>${project.version}</version>
+          <type>zip</type>
+          <optional>true</optional>
```

Comment:
@Cousjava it excludes the transitive dependencies of payara-api-package (note that cascading applies in transitive dependencies), so it only fetches one zip file and extract it.